servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpRequestFilter.java

```diff
@@ -54,17 +63,42 @@
 public final class OpenTelemetryHttpRequestFilter extends AbstractOpenTelemetryFilter
```

Comment:
I wonder if we should rename this to `OpenTelemetryHttpClientFilter` to have proper alignment with the service?